### ISSUE TITLE
Unable to select quest rewards to complete quests

### ISSUE THREAD

**Issue by JCol0208** [dalaranwow/dalaran-wow#5335]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
With any character I create I am unable to complete any quests by selecting the reward I would like to receive. None of the items highlight when I click on them and when I click complete quest it says I need to select a reward.
**Expected behaviour**: Tell us what should happen instead
I should be able to gett my reward item and complete the quest.
**Steps to reproduce the problem**:

1.  Talk to quest return point NPC.
2.  Click reward.
3.  Nothing happens
4.  "Please select a reward."

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This is a common issue for quests where you have to collect some questitems. The issue is related to a server restart.

Please remove the quest from your questlog and re-accept it.

If this does not help contact a gamemaster please. 

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
